app/src/main/java/com/practicum/playlistmaker/TrackAdapter.kt

```diff
@@ -0,0 +1,23 @@
+package com.practicum.playlistmaker
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+
+class TrackAdapter(
+    private val track: List<Track>
```

Comment:
```suggestion
    private val tracks: List<Track>
```
Так будет более точно